composer.json

```diff
@@ -17,14 +17,14 @@
   "require": {
     "php": "^7.1|^8.0",
     "firebase/php-jwt": "^v6.1.0",
-    "guzzlehttp/psr7" : "^1.2"
+    "guzzlehttp/psr7" : "^2.4.3"
```

Comment:
I believe PHP does it packages like Python where it resolves a set of dependencies (rather than like node where it can install different versions of the same library).

Given this, this would force all consumers to be using the very latest version of `guzzlehttp/psr7`. Do we want to be so restrictive? Could we allow them to use `^2.0.0` for example? And if we could, should we?

Reply:
@danielcarney96 @townxelliot Do either of you have any thoughts on this?

Hoping to crowd-source the code review on this PR slightly since we don't have any PHP specialists in-house on Notify currently. Are you both happy with this PR as it stands and think it's suitable for the wider community?

Reply:
Yeah it's a good point, it should be as unrestrictive as possible. Ideally it should be `^2.0.0` however there's security advisories on that version. The lowest i'd recommend is `^2.1.1` as there's no known security issues and it keeps it at the highest major release.

Reply:
I've updated the PR to `^2.1.1` 👍